scripts/start.js

```diff
@@ -0,0 +1,7 @@
+const execute = require('child_process').execSync
+
+const clientPort = process.env.ARAGON_CLIENT_PORT || 3000
```

Comment:
I think for now it's easiest to stay consistent and use the `REACT_APP` prefix, and then we can wholesale remove it from both this repo and aragon-cli in one go after.